packages/grafana-ui/src/themes/GlobalStyles/hacks.ts

```diff
@@ -0,0 +1,21 @@
+import { css } from '@emotion/react';
+
+export interface Hacks {
+  hackNoBackdropBlur?: boolean;
+}
+
+export function getHacksStyles(hacks: Hacks) {
+  return css([
+    /**
+     * Disables all backdrop blur effects to improve performance on extremely
+     * resource constrained devices.
+     *
+     * Controlled via the `noBackdropBlur` feature toggle in Grafana
+     */
+    hacks.hackNoBackdropBlur && {
+      '*, *:before, *:after': {
```

Comment:
considering this may be used for something else in plugins than just bluring it may be tricky, but wonder if don't want to replace with something non-blur but indicating the content is occluded, like some dimming? I assume only blur is computationally expensive.

Reply:
Yeah, it's not an ideal workaround as it targets all `backdrop-filter` properties, not just blur. But I think this'll be mostly fine, especially as an experimental feature toggle.

The MegaMenu/Modal/Drawer backdrop already has a semi-transparent background to indicate the overlay, which I think is fine for this.